src/SFA.DAS.EAS.Web/Orchestrators/EmployerManageApprenticeshipsOrchestrator.cs

```diff
@@ -364,11 +362,36 @@ public sealed class EmployerManageApprenticeshipsOrchestrator : CommitmentsBaseO
 
                 CheckApprenticeshipStateValidForChange(data.Apprenticeship);
 
-                var earliestDate = _currentDateTime.Now > _academicYearDateProvider.LastAcademicYearFundingPeriod
-                    && data.Apprenticeship.StartDate.Value < _academicYearDateProvider.CurrentAcademicYearStartDate
-                    ? _academicYearDateProvider.CurrentAcademicYearStartDate
-                    : data.Apprenticeship.StartDate.Value;
-                    
+                
+                DateTime earliestDate = data.Apprenticeship.StartDate.Value;
+
+                if (changeType == ChangeStatusType.Resume && data.Apprenticeship.StartDate.Value > _currentDateTime.Now)
+                {
+                    earliestDate = data.Apprenticeship.PauseDate.Value;
+                }
+                else if (changeType == ChangeStatusType.Resume)
+                {
+                    if (data.Apprenticeship.PauseDate.HasValue)
+                    {
+                        earliestDate = data.Apprenticeship.PauseDate.Value;
+                        if (data.Apprenticeship.PauseDate.Value < _academicYearDateProvider.CurrentAcademicYearStartDate)
```

Comment:
It probably is worth using the academic year validator to handle this logic. It will combine this and the next condition in one.

Reply:
Yep, in progress